_build/data/transport.core.system_settings.php

```diff
@@ -1009,6 +1009,15 @@
   'area' => 'manager',
   'editedon' => null,
 ], '', true, true);
+$settings['enable_template_picker_in_tree']= $xpdo->newObject(modSystemSetting::class);
+$settings['enable_template_picker_in_tree']->fromArray([
+    'key' => 'enable_template_picker_in_tree',
+    'value' => true,
```

Comment:
I made the default value `true` but we can change that if the consensus is that it shouldn't be enabled on install.

Reply:
It's a great feature to have enabled by default!

Reply:
I agree, but didn't want to presume ;)